doc/extensions/zephyr/application.py

```diff
@@ -33,12 +33,15 @@ class ZephyrAppCommandsDirective(Directive):
       The default is 'cmake'.
 
     \:app:
-      if set, the commands will change directories to this path to the
-      application.
+      path to the application to build.
 
     \:zephyr-app:
-      like \:app:, but includes instructions from the Zephyr base
-      directory. Cannot be given with \:app:.
+      path to the application to build, this is an app present in the upstream
+      zephyr repository. Mutually exclusive with \:app:.
+
+    \:cd-into:
```

Comment:
does this play nicely with `:maybe-skip-config:`?

Reply:
I think so, but I will double-check